tests/clims/models/test_project.py

```diff
@@ -16,7 +15,7 @@ def setUp(self):
     def test_can_create_project(self):
         project = VampireFangStudyProject(name="project1", organization=self.organization)
         project.save()
-        Project.objects.get(name=project.name)  # Raises DoesNotExist if it wasn't created
+        self.app.projects.get(name=project.name)  # Raises DoesNotExist if it wasn't created
```

Comment:
I would like it to return None if it doesn't exist!

Reply:
I suggest we keep the API as it is for now. But later, that might be what we want.

Reply:
ok